.babelrc

```diff
@@ -0,0 +1 @@
+{ "stage": 1 }
```

Comment:
`babelify` is only intended for demos that using ES6 syntax, the module itself (`index.js`) is written in pure ES5 so no transformation is needed.

But I guess it's been a long time since I released this repo initially, the tool chains changed a lot, maybe it worth upgrading to state of the art.